install.php

```diff
@@ -109,19 +109,24 @@
 
 	if (empty($message))
 	{
-		//Create config
+		$fullname = $_POST['surname'] . " " . $_POST['name'];
```

Comment:
This... Doesn't seem quite right. If I would be setting server status for a company I wouldn't want my name to be there but the company name instead. Could this be added as a separate field to the install script? Adding a constant for it would be a good idea.

Reply:
Ah great catch, yeah I'll create a seperate field.